ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -1129,14 +1135,27 @@ export default class ConfirmTransactionBase extends Component {
     const isContractInteractionFromDapp =
       (isTokenApproval || isContractInteraction) &&
       txData.origin !== 'metamask';
+
+    // TODO: swapApproval is exclusively used for swap+send currently,
```

Comment:
To clarify, we use this in both `ui/ducks/send/send.js` and `ui/ducks/swaps/swaps.js`?